firebase_admin/__init__.py

```diff
@@ -307,3 +309,5 @@ def _cleanup(self):
                 if hasattr(service, 'close') and hasattr(service.close, '__call__'):
                     service.close()
             self._services = None
+
+App.update
```

Comment:
Is this a typo?

Reply:
It is. Just pushed an update to my branch. I did run the unit tests before, all passed.